docs/source/tutorial.rst

```diff
@@ -42,11 +42,18 @@ To use a loader, (for example, ``urbansound8k``) you need to initialize it by ca
 You can indicate where the data would be stored and access by passing a path to ``data_home``, as explained below. Now ``us8k`` is a ``Dataset``
 object containing common methods, described in the following.
 
+
 Downloading a dataset
 ^^^^^^^^^^^^^^^^^^^^^
 
-All dataset loaders in soundata have a ``download()`` function that allows the user to download the :ref:`canonical <faq>`
-version of the dataset (when available). When initializing a dataset it is important to correctly set up the directory
+All dataset loaders in soundata have a ``download()`` function that allows the user to download:
+
+* The :ref:`canonical <faq>` version of the dataset (when available).
```

Comment:
When is no canonical version of the dataset available?
When there are more than one version?
What happens then, will the latest version be downloaded automatically, or is the user forced to specify a version?

Reply:
Ok, thanks! Added documentation and examples about these issues. To better support the use of different versions, I did add a `list_dataset_versions()` function in `soundata.__init__` which should help users get the available versions before initializing the data loaders. This is documented as well. This is all in my last commit.

Does that address the issue @hagenw? Please @tanmayy24 and @guillemcortes also take a look to see if that sounds good to you.

Reply:
Thanks, for updating the docs accordingly.
Looks fine to me, I just added a few small comments.